test/dialect/test_sybase.py

```diff
@@ -28,11 +28,18 @@ def test_extract(self):
                 'SELECT DATEPART("%s", t.col1) AS anon_1 FROM t' % subst,
             )
 
+    def test_limit_offset(self):
+        stmt = select([1]).limit(5).offset(6)
+        assert stmt.compile().params == {"param_1": 5, "param_2": 6}
+        self.assert_compile(
+            stmt, "SELECT 1 ROWS OFFSET :param_1 LIMIT :param_2 "
```

Comment:
Just realizing this ... the limit and offset are reversed here for some reason.

Reply:
That's odd. It looks like a quirk in the test. When I actually execute the statement it comes out right:

```
2020-05-10 04:57:43,340 INFO sqlalchemy.engine.Engine SELECT 1 ROWS OFFSET ? LIMIT ? 
2020-05-10 04:57:43,340 INFO sqlalchemy.engine.Engine (6, 5)
```

Reply:
That is indeed strange. So, this led me down a rabbit hole where I was tinkering around and found that I could use the `OFFSET` without setting `LIMIT` or `TOP`. Also, I found I could move all of the `LIMIT` & `OFFSET` logic entirely to the `limit_clause`. With these changes, the tests look correct and it performs as expected on my machine when querying the database. I added it as a separate commit in case you would prefer me to remove those changes for this PR.